packages/atomic/package.json

```diff
@@ -20,12 +20,14 @@
     "dist/",
     "loader/",
     "docs/",
-    "licenses/"
+    "licenses/",
+    "headless/"
   ],
   "scripts": {
-    "build": "npm run build:locales && npm run build:stencil && npm run doc:list-assets && npm run build:storybook",
+    "build": "npm run build:locales && npm run copy:headless && npm run build:stencil && npm run doc:list-assets && npm run build:storybook",
     "build:locales": "node --max-old-space-size=4096 ./scripts/split-locales.js",
     "build:stencil": "node --max_old_space_size=6144 ./node_modules/@stencil/core/bin/stencil build",
+    "copy:headless": "ncp node_modules/@coveo/headless/ headless/",
```

Comment:
🤔 It does not need to be copied to `dist/` ?

Do we really upload everything to the CDN?

Reply:
Not at at all, it's better to add the /headless folder since we need it for the package.json (so `@coveo/atomic/headless` resolves the files correctly)

I've just added steps to copy headless in the Atomic CDN 😄